src/pages/thankyou.tsx

```diff
@@ -45,6 +50,16 @@ const ThankYou = ({ uuid, emailAddress }: ThankYouProps): ReactElement => (
         <a href="/home" role="button" draggable="false" className="govuk-button" data-module="govuk-button">
             Add another fare
         </a>
+        <a
+            href="https://publish.bus-data.dft.gov.uk/"
+            role="button"
+            draggable="false"
+            className="govuk-button govuk-button--secondary govuk-!-margin-left-3"
+            data-module="govuk-button"
+            aria-label="continue to bods"
+        >
+            Continue to BODS
+        </a>
```

Comment:
Looking at the mock for this page (https://fdbt-prototype.herokuapp.com/signpost/complete), we're missing one little bit of text below the buttons. Not sure if the feedback link needs to move below as well?